src/components/Banner/Banner.jsx

```diff
@@ -65,11 +69,27 @@ const Banner = forwardRef(
     }, [onClose]);
 
     return (
-      <aside ref={mergedRef} className={cx(className, "banner", { rtl })} aria-label={ariaLabel}>
+      <aside
+        ref={mergedRef}
+        className={cx(className, styles.banner, "banner", {
+          [styles.rtl]: rtl,
+          ["rtl"]: rtl
+        })}
+        aria-label={ariaLabel}
+      >
         <div
-          className={cx("banner--content", `image-position__${imagePosition}`, {
-            "close-button-spacing": !!renderCloseButton
-          })}
+          id={id}
+          data-testid={dataTestId || getTestId(ELEMENT_TYPES.BANNER, id)}
+          className={cx(
+            styles.bannerContent,
+            "banner--content",
+            styles[`${camelCase("image-position__" + imagePosition)}`],
```

Comment:
why not just [`imagePosition${imagePosition}`]

Reply:
it's just confusing this way

Reply:
imagePosition need to be converted to camelCase anyway, cause even if it's one word, it's starting with lowercase letter

Reply:
lets leave it for now, and discuss this in the merged branch before version 2.0